README.md

```diff
@@ -1,141 +1,146 @@
 
- 1.	In the event that either party of the dispute attempts to bribe the court, the juror agrees to flag the case for review.
- 2.	In the event that both parties attempt to bribe the court, the juror agrees to vote for dismissal of the case.
- 3.	In the event that a single party attempts to bribe the court, the juror agrees to rule in favor of the party which did not attempt to bribe the court.
+The sortition tree must be updated between terms as the proportional stake of jurors changes. Changes are queued and executed when the heartbeat function is called.
 
-The purpose of this agreement is to create a metagame with a Nash equilibrium that favors honest jurors over malicious agents and dishonest jurors attempting to influence court decisions.
+### Juror Ruling
 
-The network can set a MIN\_LIABILITY agreement for jurors collateralized with ANT, and in exchange grant juror a baseline reputation within a court. As the juror participates in disputes within the court honestly and effectively, they will accrue additional reputation. This mechanism is designed such that anyone can get reputation by collateralizing agreements, but the cost to earn reputation through honest participation is significantly cheaper.
+Once jurors have been drafted the dispute enters a deliberation phase where jurors are expected to provide a ruling in favor of one party or the other. Jurors are expected to make an independent judgement, but we assume that out of band communication between jurors is possible. For particularly nuanced cases there may even be forums and discussion threads used to discuss the details of a dispute. However, to minimize the ability of jurors to simply copy the voting behavior of other jurors we require rulings to be submitted in a two-phase commit reveal process.
 
-When a new court is initialized, there will be a small amount of reputation relative to collateral, but overtime as the court adjudicates disputes the proportion of reputation relative to collateral will increase. As the court matures, this transition makes it less and less cost effective to influence the rulings of a court by purchasing and staking collateral.
+Jurors are given a certain number of terms, called the *commitment period*, to submit a hash of their ruling. After the commit period ends, jurors are given a certain number of terms to reveal their ruling, this period is called the *reveal period*. If a juror reveals their vote prior to the reveal period, anyone can use this information to penalize the juror in exchange for a reward. If a juror fails to commit and reveal a vote by the end of the *reveal period* they are penalized.
```

Comment:
I assume there is some random salt added to the hash to prevent brute-forcing the result?

Reply:
Yes, do you think that should be specified here, or is that detail something that should be in a technical specification document?